graphene_django/rest_framework/tests/test_mutation.py

```diff
@@ -68,3 +69,47 @@ class Meta:
     model_input_type = model_input._type.of_type
     assert issubclass(model_input_type, InputObjectType)
     assert 'cool_name' in model_input_type._meta.fields
+    
+
+def test_custom_serializer(capfd):
```

Comment:
oh, got it: https://docs.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function (TIL) :)

I think you can remove it, since it not used :)